redfish_utilities/inventory.py

```diff
@@ -284,6 +284,42 @@ def catalog_resource( context, resource, inventory, chassis_id ):
                     description_str = description_str + " Storage Controller"
         catalog["Description"] = description_str.strip()
 
+    # Determine the relative repairable unit
+    relative_repairable_unit = None
```

Comment:
I think it would be good to insert the "default" value at line 200 where the catalog dictionary is first initialized for the resource for consistency in the construction of the object.

Reply:
Done